src/gc-pages.c

```diff
@@ -24,6 +24,9 @@ static size_t current_pg_count = 0;
 
 void jl_gc_init_page(void)
 {
+    if (jl_options.region_pg_cnt >= MIN_REGION_PG_COUNT) {
```

Comment:
If less than the minimum, it should probably bump it up equal to the minimum rather than ignoring it